src/pynwb/ecephys.py

```diff
@@ -19,20 +18,25 @@ class ElectrodeGroup(NWBContainer):
     __nwbfields__ = ('name',
                      'description',
                      'location',
-                     'device')
+                     'device',
+                     {'name': 'spike_event_series', 'child': False, 'doc': 'doc'})
```

Comment:
This should be an *ElectricalSeries*, and *doc* should be something that makes sense. The docstring will get rendered on the API docs, so this is what users will see if they use the API docs for reference.